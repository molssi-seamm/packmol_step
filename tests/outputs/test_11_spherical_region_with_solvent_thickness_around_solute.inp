tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside sphere 0.0 0.0 0.0 14.5277
   center
   fixed -0.0463 0.0299 0.1520 0.0 0.0 0.0
   number 1
end structure
structure input_2.pdb
   inside sphere 0.0 0.0 0.0 14.5277
   number 421
end structure
