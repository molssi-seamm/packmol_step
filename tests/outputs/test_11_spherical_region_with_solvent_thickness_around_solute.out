    Will create a spherical region containing the following molecules:

        +-------------+------------------+---------+
        |  Component  | Structure        |   Ratio |
        |-------------+------------------+---------|
        |   solute    | c1ccccc1c2ccccc2 |         |
        |    fluid    | O                |       1 |
        +-------------+------------------+---------+

    The dimensions of the region will be calculated from the solute dimensions.
    If the input structure is periodic, its dimensions will be used. Otherwise
    for molecules, a the region will be a box with extra space of 10.0 Å around
    the molecule. The number of molecules of the fluid will be obtained  by
    using the density 1.0 g/ml.

Created a spherical region with a diameter of 29.06 Å with the solute and 421.0 solvent molecules

    +-------------+------------------+---------------+----------+------------+
    |  Component  | Structure        |   Requested % |   Number | Actual %   |
    |-------------+------------------+---------------+----------+------------|
    |   solute    | c1ccccc1c2ccccc2 |               |        1 |            |
    |    fluid    | O                |       100.000 |      421 | 100.000    |
    +-------------+------------------+---------------+----------+------------+

There are a total of 1285 atoms in the cell giving a density of 1.0006 g/ml.